### PR TITLE
[Android] Update project file to setup UI Testing

### LINKED ISSUE #5916: [Android] Update project to support UI Tests

### PR DESCRIPTION
**Pull request by almedina-ms**
# Related Issue

Fixes #5916

# Description

Updates the android project to set up the environment for the development of the ui test app and tests
* Declare a new project application
* Update the build.gradle file to include the espresso library
* Create mock test that always succeed

# Sample Card

No sample card

# How Verified

Local build

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/5999)

### CODE REVIEW

File: source/android/uitestapp/src/AdaptiveCardObjectModel.i [hunk 1]
```diff
@@ -0,0 +1,217 @@
+/* Copyright (c) Microsoft Corporation. All rights reserved. */
```
Comment (paulcam206) [inline source/android/uitestapp/src/AdaptiveCardObjectModel.i:1]: Is there no way to consume the other `.i` file here? it'd be a bummer if we have to keep them both up to date...

File: source/android/uitestapp/src/androidTest/java/io.adaptivecards.uitestapp.testing/UiTests.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package io.adaptivecards.uitestapp.testing;
```
Comment (paulcam206) [inline source/android/uitestapp/src/androidTest/java/io.adaptivecards.uitestapp.testing/UiTests.java:1]: missing copyright banner

File: source/android/uitestapp/src/main/res/SimpleAdaptiveCards.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{
+	"@type": "AdaptiveCard",
```
Comment (paulcam206) [inline source/android/uitestapp/src/main/res/SimpleAdaptiveCards.json:2]: `@type`?

File: source/android/uitestapp/src/AdaptiveCardObjectModel.i (hunk 1 above)
Comment (golddove) [reply source/android/uitestapp/src/AdaptiveCardObjectModel.i:1]: +1, the object model should be included by depending on the `adaptivecards` module, right?

**Review by golddove — COMMENTED** [review]
If I understand correctly, assets are duplicated from the sample app. I believe there's a way to link assets by path, to avoid having the same asset in two locations. It would be nice to not have to maintain two copies :)

File: source/android/uitestapp/src/AdaptiveCardObjectModel.i (hunk 1 above)
Comment (golddove) [reply source/android/uitestapp/src/AdaptiveCardObjectModel.i:1]: +1, the object model should be included by depending on the adaptivecards module, right?

**Review by golddove — APPROVED** [review]

**Review by golddove — COMMENTED** [review]
Looks good

**Review by golddove — APPROVED** [review]

**Pull request merged**